utility/src/main/java/org/oppia/android/util/system/BUILD.bazel

```diff
@@ -0,0 +1,50 @@
+"""
+Utilities corresponding to simplify using Oppia Clock.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "oppia_clock",
+    srcs = [
+        "OppiaClock.kt",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+kt_android_library(
+    name = "oppia_clock_impl",
+    srcs = [
+        "OppiaClockImpl.kt",
+    ],
+    deps = [
+        ":oppia_clock",
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "oppia_clock_module",
+    srcs = [
+        "OppiaClockModule.kt",
+    ],
+    visibility = ["//visibility:public"],
```

Comment:
Reminder to update visibility per #2775 once that's merged in. Ditto for other visibilities.

Reply:
Done